enterprise/config/buildbuddy.local.yaml

```diff
@@ -13,8 +13,58 @@ storage:
     root_directory: /tmp/buildbuddy_enterprise
   enable_chunked_event_logs: true
 cache:
+  # Sample flow if migration disk -> pebble
+
+  # Begin double writing, and still return data from disk
   disk:
     root_directory: /tmp/buildbuddy-enterprise-cache
+    migration_src: true
+  pebble:
+    root_directory: /tmp/buildbuddy-enterprise-cache
+    migration_dest: true
+
+  # Copy all data from disk -> pebble
+  disk:
+    root_directory: /tmp/buildbuddy-enterprise-cache
+    migration_src: true
+  pebble:
+    root_directory: /tmp/buildbuddy-enterprise-cache
+    migration_dest: true
+  migration_cache:
+    migrate_and_exit: true
+
+  # Double read for a period, fix any bugs, still use disk as source of truth
+  disk:
+    root_directory: /tmp/buildbuddy-enterprise-cache
+    migration_src: true
+  pebble:
+    root_directory: /tmp/buildbuddy-enterprise-cache
+    migration_dest: true
+  migration_cache:
+      double_read_percentage: 0.1
```

Comment:
Could you explain what this would mean in more detail? (I have a guess but just want to make sure it's what you're thinking: basically that for 10% of reads we would read from both caches and if there was an error or diff between the results we'd report it?)

Reply:
Yep! Pseudocode is in Get and ProcessDoubleReads